sympy/functions/special/gamma_functions.py

```diff
@@ -160,6 +160,13 @@ def _eval_is_real(self):
         if x.is_positive or x.is_noninteger:
             return True
 
+    def _eval_is_positive(self):
+        x = self.args[0]
+        if x.is_positive:
+            return True
+        elif x.is_noninteger:
```

Comment:
Thanks! , that would be good, I will add them, and the second test is expected to Fail as our assumption system have this issue:
`floor(x - S.Half).is_even` returns `True` for negative even 'x' .
I have reported the issue at [here](https://github.com/sympy/sympy/issues/8853) .
Anyways we can XFAIL that.